File: PL021.java
import java.util.Scanner;

public class PL021 {
    public static void main(String[] args) {
        int n = 7;
        int key;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the key index : range(0 - " + (n - 1) + ") : ");
        key = scanner.nextInt();
        scanner.close();
        int[] arr = { 1, 2, 4, 5, 6, 7, 8 };

        if (key <= n - 1) {
            System.out.println("Element present at index " + key + " is : " + arr[key]);
        } else {
            System.out.println("Key is out of bound");
        }
    }
}

File: PL021.cpp
#include <iostream>
using namespace std;

int main()
{
    int n = 7;
    int key; // Take it from user
    cout << "Enter the key index : range(0-" << n - 1 << ") : ";
    cin >> key;
    int arr[] = {1, 2, 3, 4, 5, 5, 6};
    if (key <= n - 1)
    {
        cout << "Element found at " << key << " index is : " << arr[key] << endl;
    }
    else
    {
        cout << "Key is out of bound range" << endl;
    }
    return 0;
}